mappings/net/minecraft/SharedConstants.mapping

```diff
@@ -93,6 +99,9 @@ CLASS net/minecraft/unmapped/C_bcjitlto net/minecraft/SharedConstants
 	FIELD f_zjgkboug ENABLE_DEBUG_HOTKEYS Z
 	FIELD f_zpuntnhn USE_NEW_RENDER_SYSTEM Z
 	FIELD f_zxpaxrhe ENABLE_VILLAGE_SECTIONS_DEBUGGER Z
+	METHOD m_bkfmqder (II)Z
```

Comment:
Where is this used? Looking at the method im not quite sure, but its usage might be helpful

Reply:
maybe something to do with the new name for `DEBUG_BIOME_SOURCE`? I see you already looked at the mojmap name with no luck

Reply:
Maybe `isSquareTerrain`?

Reply:
`DEBUG_BIOME_SOURCE` was probably a mismatch ported in #10. I'd say `isVoidTerrain`, because generators and carvers seem to check it returns false before carving/generating

Reply:
huh ok. I dont see it there. might have been intermediary. `isVoidTerrain` sounds good